### PR TITLE
Update RightTriangle.cpp

### PR DESCRIPTION
**Pull request by mafijas13**

### CODE REVIEW

**Comment by mafijas13** [comment]
plz check not sure if i did those two right.

File: RightTriangle.cpp [hunk 1]
```diff
@@ -1,11 +1,40 @@
...
+RightTriangle::RightTriangle()
 {
+	this->a.setPoint(0,0);
+	this->b.setPoint(0,0);
+	this->c.setPoint(0,0);
+
 }
+RightTriangle::RightTriangle(GenPoint vertex, int height, int base, Color color)
+{
+	vertex.setPoint(a.getX(),a.getY());
```
Comment (christmoore) [inline RightTriangle.cpp:21]: Needs to be b.getX(), b.getY()

File: RightTriangle.cpp [hunk 2]
```diff
@@ -1,11 +1,40 @@
...
+	
+
+	b.setPoint(a.getX(),a.getY()+height);
```
Comment (christmoore) [inline RightTriangle.cpp:24]: Needs to be a.getX(), b.getY()-height

File: RightTriangle.cpp [hunk 3]
```diff
@@ -1,11 +1,40 @@
...
+	c.setPoint(a.getX()+base,a.getY());
```
Comment (christmoore) [inline RightTriangle.cpp:25]: Needs to be c.getX()+base, c.getY()

**Pull request closed without merging**